src/discord.ts

```diff
@@ -95,6 +95,12 @@ discord.on(
 );
 
 discord.on("message", async (message: Message) => {
+  if(message.content.includes(`<@!${config.discord.whitep4nth3rId}>`)) {
+    if(message.author.id !== config.discord.p4nth3rb0tId) {
+      message.reply(`you are fined one credit for a violation of the Verbal Morality Statute.\n\nPlease refrain from mentioning whitep4nth3r directly if it is not extremely urgent. Thank you.`);
```

Comment:
Can we tag the person with maybe "Attention, @user!" and give the "you" a capital letter please?

Reply:
Oh but that won't work with the reply maybe. Disregard if nonsense.

Reply:
The reply method prefixes with `@user,` which is why it starts that way. I could ditch the reply and create a new message with `Attention, @user! You are fined....` if you prefer it?

Reply:
I could probably get rid of the meme text and display a proper warning since it is a pop culture reference?

Reply:
Yeah that might be better.